uhh.go

```diff
@@ -100,7 +159,20 @@ func (u *Uhh) Add(tag string, cmd string, searchTerms string) error {
 	}
 
 	newContents := string(contents) + fmt.Sprintf("%s\n%s\n", cmd, searchTerms)
-	ioutil.WriteFile(tagPath, []byte(newContents), os.ModePerm)
+    err = ioutil.WriteFile(tagPath, []byte(newContents), os.ModePerm)
+
+    if err != nil {
+        return fmt.Errorf("Unable to write the command and search terms: %w\n", err)
+    }
+
+    if u.config.vals.SyncOnAdd {
+        if !gitAdd(u.config) || !gitCommit(u.config) {
```

Comment:
nit: i find two negatives harder to read, prefer: `!(gitAdd(u.config) && gitCommit(u.config)`